packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -0,0 +1,128 @@
+import retrying
+
+
+LATENCY = 60
+
+
+@retrying.retry(wait_fixed=2000, stop_max_delay=LATENCY * 1000)
```

Comment:
why do we have to retry this? What is failing that makes it so we can't know deterministically what the results are?

This retry loop is also retrying if it fails for _any_ reason (including that the asserts in the test failed). If something needs to be retried we have to be far more precise. The retry time in a running cluster should also be far less than 60 seconds.